### ISSUE TITLE
Can't add new secret key

### ISSUE THREAD

**Issue by wtfred** [plesk/ext-secret-keys-manager#8]
Hello,

I can't add a new secret key by the interface. It's working by using the cli "secret_key"

Here is the error i get on the interface when i submit the form for new key:
`ERROR: Zend_Loader_PluginLoader_Exception: Plugin by name &#039;IpAddress&#039; was not found in the registry; used paths:
AdminPanel_Validate_: AdminPanel/Validate/
Plesk\CommonPanel\Validate\: CommonPanel/Validate/
CommonPanel_Validate_: CommonPanel/Validate/
Zend_Validate_: Zend/Validate/ (PluginLoader.php:424)`

When commenting the 3 lines added in this commit : https://github.com/plesk/ext-secret-keys-manager/commit/acd332813309d691c888ff9d12f83c10da377614 i can add secret key again

Keychain for API Secret Keys Version 2.0.1-1
OS: Ubuntu 20.04.3 LTS
Product: [Plesk Obsidian 18.0.42 Update #1](), last updated on Mar 17, 2022 06:27 AM

Logs from `/var/log/plesk/panel.log`

```[2022-03-24 08:49:58.948] 2794149:623c22a6e79f0 ERR [panel] Plugin by name 'IpAddress' was not found in the registry; used paths:
AdminPanel_Validate_: AdminPanel/Validate/
Plesk\CommonPanel\Validate\: CommonPanel/Validate/
CommonPanel_Validate_: CommonPanel/Validate/
Zend_Validate_: Zend/Validate/:
0: /opt/psa/admin/plib/vendor/plesk/zf1/library/Zend/Loader/PluginLoader.php:424
        Zend_Loader_PluginLoader->load(string 'IpAddress')
1: /opt/psa/admin/plib/vendor/plesk/zf1/library/Zend/Form/Element.php:2152
        Zend_Form_Element->_loadValidator(array)
2: /opt/psa/admin/plib/vendor/plesk/zf1/library/Zend/Form/Element.php:1305
        Zend_Form_Element->getValidators()
3: /opt/psa/admin/plib/vendor/plesk/zf1/library/Zend/Form/Element.php:1403
        Zend_Form_Element->isValid(string '172.16.15.181', array)
4: /opt/psa/admin/plib/vendor/plesk/zf1/library/Zend/Form.php:2295
        Zend_Form->isValid(array)
5: /opt/psa/admin/plib/pm/Form/Simple.php:49
        pm_Form_Simple->isValid(array)
6: /opt/psa/admin/plib/modules/secret-keys-manager/controllers/IndexController.php:86
        IndexController->createAction()
7: /opt/psa/admin/plib/vendor/plesk/zf1/library/Zend/Controller/Action.php:516
        Zend_Controller_Action->dispatch(string 'createAction')
8: /opt/psa/admin/plib/vendor/plesk/zf1/library/Zend/Controller/Dispatcher/Standard.php:308
        Zend_Controller_Dispatcher_Standard->dispatch(object of type Zend_Controller_Request_Http, object of type Zend_Controller_Response_Http)
9: /opt/psa/admin/plib/vendor/plesk/zf1/library/Zend/Controller/Front.php:954
        Zend_Controller_Front->dispatch()
10: /opt/psa/admin/plib/pm/Application.php:87
        pm_Application->run()
11: /opt/psa/admin/htdocs/modules/secret-keys-manager/index.php:5
[2022-03-24 08:49:58.949] 2794149:623c22a6e79f0 ERR [panel] Plugin by name 'IpAddress' was not found in the registry; used paths:
AdminPanel_Validate_: AdminPanel/Validate/
Plesk\CommonPanel\Validate\: CommonPanel/Validate/
CommonPanel_Validate_: CommonPanel/Validate/
Zend_Validate_: `Zend/Validate/`
```

Is it a bug from your extension or do we need to submit an issue in Plesk? Thanks

**Comment by wtfred** [comment]
Problem still here with Plesk Obsidian 18.0.44 Update #3, last updated on June 17, 2022 06:26 AM
Would be nice if someone could look into it, maybe @sibprogrammer ?
Thanks

**Comment by sibprogrammer** [comment]
Thank you for the bug report. One needs to wait a little bit when a new version of extension will be released.

**Comment by wtfred** [comment]
i confirm it's fixed with 2.0.2, thanks!
